cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,9 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Instance().ReadInstanceData()
```

Comment:
we can for example create subdirectories of _~/.minishift/config_. _~/.minishift/config/config.json_ being the config file applying to all instances, opposed to _~/.minishift/config/\<machine name\>_ being the instance specific configuration. Seems logical/consistent to me as well. WDYT?